ownCloudAppShared/Client/File Lists/QueryFileListTableViewController.swift

```diff
@@ -72,6 +75,7 @@ open class QueryFileListTableViewController: FileListTableViewController, SortBa
 
 	public init(core inCore: OCCore, query inQuery: OCQuery) {
 		query = inQuery
+		searchScope = .global
```

Comment:
shouldn't this var set to `.local`. I found `.local` as default below as initial variable value and in an other source file as default value?

Reply:
That line is indeed not needed and even could overwrite the last user setting. I removed it. Thanks!